.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
+  - parts/node
+env:
+  matrix:
+    - PLONE_VERSION=4.3
+    - PLONE_VERSION=5.0
+#    - PLONE_VERSION=5.1
+before_install:
+ - virtualenv .
+ - bin/pip install --upgrade pip setuptools zc.buildout coverage==4.2
+ - if [ $PLONE_VERSION == 5.0 ]; then pip install --upgrade setuptools==21.0.0; fi
+install:
+ - sed -ie "s#test-4.3#test-$PLONE_VERSION#" buildout.cfg
+ - bin/buildout annotate
+ - bin/buildout
+script:
+ - coverage run bin/test
```

Comment:
that should be bin/coverage maybe? currently travis fails here saying that it can not be found. Probably is due to the virtualenv created at the beginning that is *not* activated (i.e. ``source  bin/activate``)